selfdrive/ui/qt/setup/setup.cc

```diff
@@ -287,6 +290,61 @@ QWidget * Setup::download_failed() {
   return widget;
 }
 
+QWidget * Setup::download_invalid_url(QLabel *url) {
```

Comment:
can't we reuse the failed widget for this? think it makes sense to show the URL there too anyway